docs/USING.rst

```diff
@@ -50,3 +50,38 @@ to hard-disk booting the installed system.
 This is, as the filepath states, merely an example of how you might deploy to
 physical nodes.  Existing deployment systems such as Canonical's MAAS have also
 been run succesfully.
+
+
+Usage Examples
+=================
+
+Included herewith are some examples about using some newly added cli and api calls. 
```

Comment:
Not sure this line is needed. Newli added api calls will soon be old api calls.

You can just say api and cli calls for registering a switch with HaaS.